script.js

```diff
@@ -0,0 +1,6 @@
+const getMyAge = input => {
+     const yourAge = Math.floor((Date.now() - Date.parse(input)) / 31556952000)
```

Comment:
[names]: It should be as generic as possible, so getMyAge>>>> getAge, yourAge>>>> age
[minor]: you do not need yourAge variable, >> return Math.floor((Date.now() - Date.parse(input)) / 31556952000)